pages/SearchPage.vue

```diff
@@ -1,5 +1,257 @@
+          text-white bg-blue-700 border
+          border-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300
+          dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800 rounded-md"
+          @click="nextPage"
+        >
+          <font-awesome-icon :icon="['fas', 'arrow-right']" />
+        </button>
+      </div>
+    </div>
   </div>
 </template>
+
+<script setup>
+import { faArrowDown, faArrowUp, faArrowRight, faArrowLeft } from '@fortawesome/free-solid-svg-icons'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { useAuthStore } from '../store/auth'
+import { ref, onMounted, onUnmounted, computed } from 'vue'
+
+const authModule = useAuthStore()
+library.add(faArrowDown, faArrowUp, faArrowRight, faArrowLeft)
+
+// reactive values
+const columnsDropdown = ref(false)
+const displayDropdown = ref(false)
+const pageDropdown = ref(false)
+const selectedDisplay = ref(10)
+const selectedPage = ref(1)
+// all columns selected by defualt
+const selectedColumns = ref(['Status', 'LastModifiedBy', 'NameRequestNumber', 'Names', 'ApplicantFirstName',
+  'ApplicantLastName', 'NatureOfBusiness', 'ConsentRequired', 'Priority', 'ClientNotification', 'Submitted',
+  'LastUpdate', 'LastComment'])
+
+// dropdown option values
+const displayOptions = [5, 10, 20, 50, 100]
+const results = ref(1380) // need to set this based on actual results from api
+const pageOptions = computed(() => Math.ceil(results.value / selectedDisplay.value))
+const columns = [
+  { name: 'Status', key: 'Status' },
+  { name: 'LastModifiedBy', key: 'Modified By' },
+  { name: 'NameRequestNumber', key: 'NR Number' },
+  { name: 'Names', key: 'Names' },
+  { name: 'ApplicantFirstName', key: 'Applicant First Name' },
+  { name: 'ApplicantLastName', key: 'Applicant Last Name' },
+  { name: 'NatureOfBusiness', key: 'Nature Of Business' },
+  { name: 'ConsentRequired', key: 'Consent Required' },
+  { name: 'Priority', key: 'Priority' },
+  { name: 'ClientNotification', key: 'Notified' },
+  { name: 'Submitted', key: 'Submitted' },
+  { name: 'LastUpdate', key: 'Last Update' },
+  { name: 'LastComment', key: 'Last Comment' }
+]
+
+// clear filters i.e return to orignal ref values
+
+const clearFilters = () => {
+  columnsDropdown.value = false
+  displayDropdown.value = false
+  pageDropdown.value = false
+  selectedDisplay.value = 10
+  selectedPage.value = 1
+  selectedColumns.value = ['Status', 'LastModifiedBy', 'NameRequestNumber', 'Names', 'ApplicantFirstName',
+    'ApplicantLastName', 'NatureOfBusiness', 'ConsentRequired', 'Priority', 'ClientNotification', 'Submitted',
+    'LastUpdate', 'LastComment']
+}
+
+// to close dropdown when click triggered outside of it OR when it's itself clicked
+
+const toggleDropdown = (dropdownName, event = null) => {
+  if (event) {
+    if (columnsDropdown.value && !event.target.closest('.columnsDropdown')) columnsDropdown.value = false
```

Comment:
Wait, so you don't have rich components available to you (like Vuetify)?

Reply:
Unfortunately no, not in tailwind.

Reply:
How about creating your own (small) component library? You could create a component out of this code, and use it here.